README.md

```diff
@@ -1,2 +1,33 @@
-# HelloWorld
-First
+
+
+Project Name:-HelloWorld
+
+Description the project:-First Microverse project
+
+Built With
+
+Major languages:-Html and Css
+Frameworks:-Vsc
+
+👤 Author1:-Yilikal Teshome
+
+GitHub: @Yilkal250
+Twitter: @Yilikal250
+LinkedIn: teshomeyilikal250@gmail.com
+
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page. Show your support
+
+Give a ⭐️ if you like this project! Acknowledgments
+
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] Your readme file is not looking professional. All main headers and contents look similar. The headers should be bold and bigger than the content. Please fix that. You can check [this template](https://github.com/microverseinc/readme-template/blob/master/README.md) to get an idea.